src/ucs/sys/compiler.h

```diff
@@ -72,6 +72,13 @@
  */
 #define ucs_unaligned_ptr(_ptr) ((void*)(_ptr))
 
+/**
+ * suppress compilation warning on string size on gcc 8+
+ */
+#define ucs_safe_strncpy(_dst, _src, _len) \
```

Comment:
should be named ucs_strncpy_safe (for example we have ucs_ffs64_safe and ucs_list_for_each_safe)

Reply:
fixed